service/service.py

```diff
@@ -198,6 +198,30 @@ def update_customers(customer_id):
     app.logger.info("Customer with ID [%s] updated.", customer.id)
     return make_response(jsonify(customer.serialize()), status.HTTP_200_OK)
 
+
+######################################################################
+# SUSPEND AN EXISTING CUSTOMER
+######################################################################
+@app.route("/customers/<int:customer_id>", methods=["PUT"])
```

Comment:
The endpoint would probably look different: something like /customers/<int:customer_id>/suspend
 Looked at slide 59 of 04-Building-RESTful-Services.pdf